frontend/src/components/Giphy.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from 'react'
+
+import styled from "styled-components/macro"
+
+const apiKey = process.env.REACT_APP_API_KEY
+console.log(`apiKey: ${apiKey}`)
```

Comment:
PS. Don't log your API-key, you have it as an environmental variable to keep it from the client. Also, check so it's set up correctly both in the .env file and on netlify, because I'm getting undefined on the gif :/ The URL looks like it has quotes in it, read more here if you'd like: [https://www.w3schools.com/tags/ref_urlencode.ASP](https://www.w3schools.com/tags/ref_urlencode.ASP).